src/main/scala/firrtl/LoweringCompilers.scala

```diff
@@ -150,7 +150,8 @@ class EmitVerilogFromLowFirrtl (val writer: Writer) extends Transform with Simpl
       passes.SplitExpressions,
       passes.CommonSubexpressionElimination,
       passes.DeadCodeElimination,
-      passes.VerilogRename)
+      passes.VerilogRename,
+      passes.VerilogPrep)
```

Comment:
There has been talk of rewriting the Verilog Emitter as a sequence of Firrtl passes, and then a very simple Verilog Emitter (as the Firrtl graph is already in a Verilog-esque state). This is me starting that by simplifying the Instance emission (which was also required to emit inout properly)